Source/WebKit/Shared/Extensions/WebExtensionUtilities.h

```diff
@@ -113,6 +113,14 @@ id toWebAPI(const std::optional<T>& result, UseNullValue useNull = UseNullValue:
     return toWebAPI(result.value());
 }
 
+inline NSDictionary *toWebAPI(const JSON::Object& object)
+{
+    String jsonString = object.toJSONString();
+    auto *jsonData = [(NSString *)jsonString dataUsingEncoding:NSUTF8StringEncoding];
+    NSDictionary *dictionary = [NSJSONSerialization JSONObjectWithData:jsonData options:NSJSONReadingAllowFragments error:nil];
```

Comment:
I think an implicit `jsonString` will work for the method call, no need to cast, IIRC.
```suggestion
    auto jsonString = object.toJSONString();
    auto *jsonData = [jsonString dataUsingEncoding:NSUTF8StringEncoding];
    auto *dictionary = [NSJSONSerialization JSONObjectWithData:jsonData options:NSJSONReadingAllowFragments error:nil];
```